### PR TITLE
upgrade to SBT version 1.1.0

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

File: project/scripted.sbt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-libraryDependencies <+= (sbtVersion) { sv =>
+libraryDependencies += (sbtVersion) { sv =>
   "org.scala-sbt" % "scripted-plugin" % sv
```
Comment (eed3si9n) [inline project/scripted.sbt]: I think this needs to be change to %%.

**Comment by sullis** [comment]
there is 1 remaining error. Can you take a look?  @eed3si9n eed3si9n

**Comment by sullis** [comment]
there is 1 remaining error.  @eed3si9n  can you take a look?

**Comment by eed3si9n** [comment]
Sure. I'll take a look this weekend.

**Pull request merged**